### PR TITLE
fix: [#1367] fix db oag scheme

### PR DESCRIPTION
**Pull request by wiktorflorian**

### CODE REVIEW

**Review by Michal-Kolomanski — CHANGES_REQUESTED** [review]

File: transform/schemas/common/oag/instance.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ class Instance(BaseModel):
     full_text: Optional[str]
     hosted_by: Optional[KeyValueModel]
     license: Optional[str]
-    measures: Optional[List[KeyValueModel]]
+    measures: Optional[List[KeyValueModel]]  # TODO: check if this exists
```
[the remark this answers is not in the document]
Comment (wiktorflorian) [reply transform/schemas/common/oag/instance.py:51]: Deleted

File: transform/schemas/db/dataset.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 """Dataset expected db schema"""
+# TODO: implement connection between organisation, author, collected_from, context, instance, projects, relations and subject
```
[the remark this answers is not in the document]
Comment (wiktorflorian) [reply transform/schemas/db/dataset.py:3]: ok, removed

File: transform/schemas/db/oag/__init__.py
[the remark this answers is not in the document]
Comment (wiktorflorian) [reply transform/schemas/db/oag/__init__.py:1]: that's right

**Review by Michal-Kolomanski — APPROVED** [review]
:clap

**Pull request merged**